aplib.net-demo/Assets/Testing/AplibTests/RangedEnemyTest.cs

```diff
@@ -0,0 +1,139 @@
+using Aplib;
+using Aplib.Core;
+using Aplib.Core.Belief;
+using Aplib.Core.Desire;
+using Aplib.Core.Desire.Goals;
+using Aplib.Core.Intent.Actions;
+using Aplib.Core.Intent.Tactics;
+using Aplib.Integrations.Unity.Actions;
+using NUnit.Framework;
+using System.Collections;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+using UnityEngine.TestTools;
+
+namespace Testing.AplibTests
+{
+    public class RangedEnemyBeliefSet : BeliefSet
+    {
+        public static readonly int FramesToRemember = 300;
```

Comment:
Seems like a constant?
```suggestion
        public const int FramesToRemember = 300;
```